cumulus_library/template_sql/statistics/count.sql.jinja

```diff
@@ -48,8 +72,14 @@ CREATE TABLE {{ table_name }} AS (
             {%- if secondary %}
             {{ secondary }}_ref,
             {%- endif -%}
+            {%- if fhir_resource=='condition' %}
+            coalesce(cast(cond_code_display AS varchar), 'missing-or-null') AS cond_code_display,
```

Comment:
I made this little helper method, because I didn't like reproducing a static string all over, which might get typo'd. It doesn't save much, though, so I get if you like the cleanliness of just doing it directly. But here is my macro:

```
{% macro coalesce_missing() -%}
    {% if varargs %}
        {% set arg = varargs[0] %}
    {% else %}
        {% set arg = caller() %}
    {% endif %}

    COALESCE(
        {{ arg }},
        'missing-or-null'
    )
{%- endmacro %}
```

Can be used like so:
```
 {{ coalesce_missing('field.subfield') }} AS cond_code_display,
 {% call coalesce_missing() %}
    cast(cond_code_display AS varchar)
 {% endcall %} AS cond_code_display
```

Reply:
I think the macro idea is smart, especially as we're about to change this based on some of the system discussion I was mentioning earlier - would be nice to have one point of entry for this.

That said - I might elect to do that in the next PR so I have a better idea of the expected scopes - lemme sleep on this one and make a call one way or another.